include/krml/internal/target.h

```diff
@@ -57,6 +57,18 @@
 #  define KRML_HOST_IGNORE(x) (void)(x)
 #endif
 
+#ifndef KRML_NOINLINE
+#  if defined(_MSC_VER)
```

Comment:
Is it true that LLVM also sets _MSC_VER on windows? Maybe we need some more flags conditions here?

Reply:
I'm not sure, but all the macros in this file follow the convention that _MSC_VER means "MSCV-compatible" (and presumably, should clang define this macro, then clang would be able to parse __declspec rather than __attribute__), and __GNUC__ means gcc/clang...

If that's not right then we have a bigger problem than just this macro